apps/ui-builder/src/components/ComponentList.tsx

```diff
@@ -143,6 +160,16 @@ export const ComponentList: React.FunctionComponent<ListProps> = ({ onDragStart,
           </AccordionItem>
         ))}
       </Accordion>
+      {recommendedComponentsList.length > 0 && (
```

Comment:
this works very well! to make it easier to discover, could you please move the recommended components inside of the accordion to the beginning? it would then land before
```
{treeItems.map(c => (
```
inside of `<Accordion>` - and would render AccordionItem, AccordionHeader, AccordionPanel and spans in it.

Reply:
Thanks for the suggestion, I moved the category inside of the accordion component